tests/integration_tests/validation/validation_util.py

```diff
@@ -378,6 +379,42 @@ def load_timing_file(name: str) -> dict[str, float]:
         }
 
 
+def save_actual_results_file(config: ConformanceSuiteConfig, results: list[ParameterSet], diff: bool = True) -> None:
```

Comment:
nit: this function is doing more than the name implies (it's also maybe creating and saving a diff).

Reply:
Good point, will update.